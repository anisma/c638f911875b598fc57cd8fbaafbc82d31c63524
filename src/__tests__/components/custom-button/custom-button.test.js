import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import CustomButton from '../../../components/custom-button/custom-button.component'

describe('CustomButton Component', () => {
  it('renders correctly enzyme', () => {
  const wrapper = shallow(<CustomButton />)

  expect(toJson(wrapper)).toMatchSnapshot();
});
});