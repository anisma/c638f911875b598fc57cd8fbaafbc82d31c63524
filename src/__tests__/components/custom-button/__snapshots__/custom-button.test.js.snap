// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomButton Component renders correctly enzyme 1`] = `<styled.button />`;
